.github/workflows/linux.yml

```
@@ -47,12 +47,14 @@ jobs:
         env:
           CI_NETWORK: true
           VERSION: ${{ needs.library.outputs.version }}
+          PYTHONWARNINGS: "ignore::DeprecationWarning:gi.events"
         run: |
```

actionlint (1.7.12)

[error] object type "{arch: string; container: string; distro: string; variant: string}" cannot be filtered by object filtering `.*` since it has no object element

(expression)